### PR TITLE
Rhmap 12487

### PR DESCRIPTION
**Pull request by pb82**
# Motivation

calls the new `cleanup` endpoint in fh-mbaas to perform temp directory cleanup

### CODE REVIEW

File: lib/admin/appdata/cleanup.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+var mbaasRequest = require('../../mbaasRequest/mbaasRequest')
+  , constants = require('../../config/constants')
+  , config = require('../../config/config');
+
+/**
+ * Start a new tmp data cleanup job
+ * @param numNodes The number of nodes this cluster has
+ */
+function startJob(params, cb) {
+  var resourcePath = config.addURIParams(constants.CLEANUP_BASE_PATH, params);
```
Comment (rachael-oregan) [inline lib/admin/appdata/cleanup.js]: Same here, `const` instead of `var`?

File: lib/admin/appdata/cleanup.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+var mbaasRequest = require('../../mbaasRequest/mbaasRequest')
```
Comment (pb82) [inline lib/admin/appdata/cleanup.js]: yes, thats right. Will change it.

File: lib/admin/appdata/cleanup.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  , constants = require('../../config/constants')
+  , config = require('../../config/config');
+
+/**
+ * Start a new tmp data cleanup job
+ * @param numNodes The number of nodes this cluster has
```
Comment (sedroche) [inline lib/admin/appdata/cleanup.js]: Is `@param {Number} params.numNode - The number of nodes this cluster has`  better?

File: package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "fh-mbaas-client",
   "description": "FeedHenry MBaaS Client",
-  "version": "0.16.5",
```
Comment (jcesarmobile) [inline package.json]: shouldn't be 0.17.0?

Comment (pb82) [inline package.json]: yes thats right, i added something. Will change it.

**Pull request closed without merging**